pkg/gateway/gateway.go

```diff
@@ -53,9 +55,11 @@ import (
 )
 
 const (
-	defaultLeaseDuration = 10 * time.Second
-	defaultRenewDeadline = 5 * time.Second
+	defaultLeaseDuration = 15 * time.Second
```

Comment:
@yboaron to me these values look reasonable and are aligned with what K8s leaderElection code recommends. But if you have any concerns, do let us know.